MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -394,6 +408,10 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
 					_texCoordTL, 
 					_texCoordBR);			
 			
+		    unchecked
+		    {
+                ++_drawCount;
```

Comment:
However, maybe you right... this is nice looking and nobody knows how it is looking in C# assembly code.